### PR TITLE
SPCTR-1452 - fix: UI/UX issues reported by Designers in Container.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
https://brainstormforce.atlassian.net/browse/SPCTR-1452?atlOrigin=eyJpIjoiMzJmNmEyMmJlYzczNGJhOWIzYTIxYmZmY2RmNWE5YWEiLCJwIjoiaiJ9

### Types of changes
 Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - In the card.

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/container/settings.js [hunk 1]
```diff
@@ -1678,7 +1694,19 @@ const Settings = ( props ) => {
...
 							max={ 500 }
-							displayUnit={ false }
+							unit={ {
+								value: topDividerHeightType,
+								label: 'topDividerHeightType',
+							} }
+							units={ [
+								{
+									name: __(
+										'px',
```
Comment (AbhijitNage123) [inline src/blocks/container/settings.js:1704]: name should be capital  for unit PX

File: src/blocks/container/settings.js [hunk 2]
```diff
@@ -1796,7 +1836,19 @@ const Settings = ( props ) => {
...
 							max={ 500 }
-							displayUnit={ false }
+							unit={ {
+								value: bottomDividerHeightType,
+								label: 'bottomDividerHeightType',
+							} }
+							units={ [
+								{
+									name: __(
+										'px',
```
Comment (AbhijitNage123) [inline src/blocks/container/settings.js:1846]: name should be capital for unit PX

File: src/blocks/container/attributes.js [hunk 3]
```diff
@@ -969,6 +969,22 @@ const attributes = {
 		type: 'string',
 		default: 'visible'
 	},
+	topDividerWidthType: {
+		type: 'string',
+		default: '%',
```
Comment (AbhijitNage123) [inline src/blocks/container/attributes.js:970]: UAGCopyPaste missing for newly added attribute

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

File: src/blocks/container/attributes.js (hunk 3 above)
Comment (RajkiranBagal) [reply src/blocks/container/attributes.js:970]: It's just for functional purpose & is not used anywhere. No need to add Copy Paste here.

File: src/blocks/container/settings.js (hunk 1 above)
Comment (RajkiranBagal) [reply src/blocks/container/settings.js]: Updated.

File: src/blocks/container/settings.js (hunk 2 above)
Comment (RajkiranBagal) [reply src/blocks/container/settings.js]: Updated.

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**